pkg/configmap/configmap.go

```diff
@@ -50,17 +53,32 @@ func (l *BundleLoader) Load(cm *corev1.ConfigMap) (bundle *api.Bundle, err error
 	return
 }
 
-func loadBundle(entry *logrus.Entry, data map[string]string) (bundle *api.Bundle, skipped map[string]string, err error) {
+func loadBundle(entry *logrus.Entry, cm *corev1.ConfigMap) (bundle *api.Bundle, skipped map[string]string, err error) {
 	bundle = &api.Bundle{Object: []string{}}
 	skipped = map[string]string{}
 
+	gzipped := hasGzipEncodingAnnotation(cm)
+	if gzipped {
+		entry.Info("Decoding gzip-encoded bundle data")
```

Comment:
Does this also apply to the catalog-operator? Since this code is actually invoked from there during steps generation. If so, this could definitely be reduced to debug level.